docs/sources/panels/inspect-panel.md

```diff
@@ -0,0 +1,87 @@
++++
+title = "Inspect a panel"
+type = "docs"
+[menu.docs]
+identifier = "inspect-a-panel"
+parent = "panels"
+weight = 400
++++
+
+# Inspect a panel
+
+> **Note:** This documentation refers to a feature only available in Grafana 7.0 beta.
+
+The panel inspector helps you understand and troubleshoot your panels. You can inspect the raw data for any Grafana panel, export that data to a comma-separated variable (CSV) file, view query requests, and export panel and data JSON.
+
+## Panel inspector UI
+
+The panel inspector displays Inspect: <NameOfPanelBeingInspected> at the top of the pane. Click the arrow in the upper right corner to expand or reduce the pane.
+
+The panel inspector consists of four tabs:
+
+* **Data tab -** Shows the raw data returned by the query with transformations applied. Field options such as overrides and value mappings are not applied.
+* **Stats tab -** Shows how long your query takes and how much it returns.
+* **JSON tab -** Allows you to view and copy the panel JSON, panel data JSON, and data frame structure JSON. This is useful if you are provisioning or administering Grafana. 
+* **Query tab -** Shows you the requests to the server sent when Grafana queries the data source. 
+
+> **Note:** Not all panel types include all four tabs. For example, dashboard list panels do not have raw data to inspect, so they do not display the Stats, Data, or Query tabs.
+
+## Panel inspector tasks
+Tasks you can perform in the panel inspector are described below. 
+
+### Open the panel inspector
+
+You can inspect any panel that you can view.
+
+1. In Grafana, navigate to the dashboard that contains the panel you want to inspect.
+1. Click the title of the panel you want to inspect and then click **Inspect**.
```

Comment:
```suggestion
2. Click the title of the panel you want to inspect and then click **Inspect**.
```

Reply:
This is called "lazy numbering." It is what is recommended for numbered lists in Markdown.